handler/http/client/client.go

```diff
@@ -62,6 +63,16 @@ func (h *Handler) sendRequestWithBody(resourceName, target string, content *gher
 	return r.Request(tt[0], tt[1], requestBody)
 }
 
+func (h *Handler) sendRequestWithBodyFromFile(resourceName, target string, file string) error {
+	r, ok := h.r[resourceName]
+	if !ok {
+		return fmt.Errorf("%s not found", resourceName)
+	}
+
+	tt := strings.Split(target, " ")
+	return r.RequestFromFile(tt[0], tt[1], file)
```

Comment:
this would cause panic if `len(tt) < 2`, please check the len

Reply:
Copied this from other locations, I'll fix all together in one PR